sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/ConstantFoldingSuite.scala

```diff
@@ -425,3 +447,14 @@ case class SerializableBoxedInt(intVal: Int) {
 class NotSerializableBoxedInt(intVal: Int) {
   def addAsInt(other: Int): Int = intVal + other
 }
+
+case object CharLength extends ScalarFunction[Int] {
```

Comment:
Could we reuse the `CharLength` defined at `JDBCV2Suite`?

Reply:
It is located in `sql/core` module.

Reply:
It doesn't matter. You can move it from catalyst to core.

Reply:
The `CharLength ` is defined in `sql/core`. Do you mean to move it from `sql/core` to `sql/catalyst`?